### PR TITLE
Travis: Bump patch versions of matrix, update Bundler

### PR DESCRIPTION
**Pull request by olleolleolle**
This ~(experimental)~ PR is about getting the build to green.

- Update CI matrix versions and add Ruby 2.6.1
- Update Bundler
- Also: Drop removed Travis directive `sudo: false`.
- Also: Drop the `script` directive, which does the default thing, already (run the `default` Rake task)

## Question

Q: Is the repo being built at: https://travis-ci.org/rom-rb/rom-factory/pull_requests
A: It is _now_.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   - '[ -d coverage ] && bundle exec codeclimate-test-reporter'
 rvm:
-  - 2.5.0
-  - 2.4.2
-  - 2.3.6
-  - jruby-9.1.13.0
+  - 2.5.3
+  - 2.4.5
+  - 2.3.8
+  - jruby-9.1.17.0
```
Comment (solnic) [inline .travis.yml]: could you bump it to `jruby-9.2.5.0` and add `2.6.1` too?

Comment (olleolleolle) [reply .travis.yml]: Done and done!

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,21 @@
 language: ruby
-sudo: false
+
 services:
   - postgresql
   - mysql
 bundler_args: --without tools benchmarks
+before_install:
+  - gem install bundler -v'< 2'
```
Comment (solnic) [inline .travis.yml]: Why is this needed?

**Comment by solnic** [comment]
> Is the repo being built at: https://travis-ci.org/rom-rb/rom-factory/pull_requests

Seems like PR builds were disabled, it's fixed now.

File: .travis.yml (hunk 2 above)
Comment (olleolleolle) [reply .travis.yml]: Oh, perhaps it was just a gut-reaction to the _gray badge_ not having built without an error. I'll try removing this.

Comment (olleolleolle) [reply .travis.yml]: Ah, I remebered wrong: the issue is that the shipped version of Bundler can be too early to understand the platform name "truffleruby".

MRIs 2.3.8, 2.4.5, 2.5.3 come with Bundler 1.16.2.

Comment (olleolleolle) [reply .travis.yml]: I got lucky with 2.

**Comment by olleolleolle** [comment]
@solnic Would you want me to squash commits?

**Comment by solnic** [comment]
> @solnic Would you want me to squash commits?

yes please 😄

**Comment by olleolleolle** [comment]
@solnic Happy Monday to you!

**Comment by solnic** [comment]
@olleolleolle happy Monday to you too 😄 this is merged, thank you! ❤️
